protos/flyteidl/admin/workflow.proto

```diff
@@ -55,6 +56,9 @@ message WorkflowSpec {
     // propeller compiler (since the compiler doesn't have any knowledge of other workflows - ie, it doesn't reach out
     // to Admin to see other registered workflows).  In fact, subworkflows do not even need to be registered.
     repeated core.WorkflowTemplate sub_workflows = 2;
+
+    // Represents the specification for Description.
```

Comment:
can you add a comment here that logic for listing/getting will masked?  i think we won't return the long description when listing right? only the short description.  Also I think for workflows specifically if they have subworkflows we might mask more stuff too, and users have to call the get description entity to get more.

Reply:
`WorkflowSpec` and `TaskSpec` are used only when creating a workflow and task. Therefore, we send both task/workflow proto and description proto to admin when we call register task/workflow.

I just added a short description to the [task](https://github.com/flyteorg/flyteidl/blob/5b4d98ca4a5280f1cae8b7ba45d91c74add87482/protos/flyteidl/admin/task.proto#L32-L38) and [workflow](https://github.com/flyteorg/flyteidl/blob/5b4d98ca4a5280f1cae8b7ba45d91c74add87482/protos/flyteidl/admin/workflow.proto#L31-L37), and we'll use it when we call get task and list task.

Reply:
oh right yes.  thanks.